requirements/requirements-app.txt

```diff
@@ -1,36 +1,36 @@
-asyncpg==0.21.*
-attrs==20.*
-boto3==1.16.*
-certifi==2020.12.5
-dataclasses-json==0.5.*
-ddtrace==0.46.0
-dj-database-url==0.5.0
-django-cors-headers==3.11.*
-django-debug-toolbar==3.4.*
-django-extensions==3.1.*
-django-spaghetti-and-meatballs==0.4.0
-Django==3.2.*
-django_cte==1.2.*
-djangorestframework==3.13.*
-docutils==0.15.2
-drf-api-tracking==1.8.0
-drf-extensions==0.7.*
-elasticsearch-dsl==7.1.0
-elasticsearch==7.1.0
-et-xmlfile==1.0.1
-filelock==3.0.12
-fiscalyear==0.2.0
-Markdown<3.0
-marshmallow==3.14.1
-numpy==1.20.*
-openpyxl==2.4.7
-pandas==1.3.*
-psutil==5.6.*
-psycopg2-binary==2.8.*
-py-gfm==0.1.4
-pydantic[dotenv]==1.9.0
-python-json-logger==0.1.9
-requests==2.25.*
-retrying==1.3.3
-urllib3==1.26.*
-xlrd3==1.0.0
+asyncpg==0.29.0
+attrs==23.2.0
+boto3==1.34.58
+certifi==2024.2.2
+dataclasses-json==0.6.4
+ddtrace==0.47.0
+dj-database-url==2.1.0
+django-cors-headers==4.3.1
+django-debug-toolbar==4.3.0
+django-extensions==3.2.3
+django-spaghetti-and-meatballs==0.4.2
+Django==4.1.13
```

Comment:
Discussed in a call recently as a backend team, but we should try and aim for `4.2.*` as that is the current LTS version: https://www.djangoproject.com/download/